### PR TITLE
Adding rpm spec for zabbix-ldap-sync

### PR DESCRIPTION
**Pull request by bilbo-the-hobbit**
Rpm spec for zabbix-ldap-sync

Signed-off-by: Mortier Benoit <benoit.mortier@opensides.be>

### CODE REVIEW

File: packaging/rpm/zabbix-ldap-sync.spec [hunk 1]
```diff
@@ -0,0 +1,63 @@
+#
+# spec file for package python-docopt
```
Comment (dnaeon) [inline packaging/rpm/zabbix-ldap-sync.spec]: Probably a copy-paste error, but this spec file is not for docopt.

File: packaging/rpm/zabbix-ldap-sync.spec [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+# Please submit bugfixes or comments via http://bugs.opensuse.org/
+#
+
+
+Name:           zabbix-ldap-sync
+Version:        0.1
+Release:        1
+Url:            https://github.com/dnaeon/zabbix-ldap-sync
+Summary:        Sync your Zabbix users with LDAP directory server
+License:        LGPL
```
Comment (dnaeon) [inline packaging/rpm/zabbix-ldap-sync.spec]: The license of this project is BSD.

Comment (bilbo-the-hobbit) [inline packaging/rpm/zabbix-ldap-sync.spec]: corrected in the new commit

Cheers

**Comment by dnaeon** [comment]
Thanks for the quick feedback!

One more question - how is this spec file related to SuSE as mentioned in the copyright? Just want to make things clear when it comes to copyright stuff.

Thanks!

**Comment by bilbo-the-hobbit** [comment]
Hello,

removed it, it s because i build on a suse box 

Cheers

**Comment by dnaeon** [comment]
Merged, thanks!

**Pull request merged**